java/client/src/main/java/glide/utils/CommandUtils.java

```diff
@@ -0,0 +1,37 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.utils;
+
+import java.lang.reflect.Array;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Stream;
+
+public class CommandUtils {
```

Comment:
Please add a short javadoc to this class. 
Consider renaming to `ArrayUtils` since these utilities could be used for _anything_.